types/googlepay/index.d.ts

```diff
@@ -180,7 +182,9 @@ declare namespace google.payments.api {
          * parameters field to see if the properties within are applicable for
          * IsReadyToPayRequest.
          */
-        allowedPaymentMethods: PaymentMethodSpecification[];
+        allowedPaymentMethods:
```

Comment:
For the purpose of the PR, I think we can leave this definition as is, and make `PaymentMethodSpecification.tokenizationSpecification` optional instead.

I'm having a discussion with the Google Pay team about how best to define this between `isReadyToPay` and `loadPaymentData`.

Reply:
From [documentation](https://developers.google.com/pay/api/web/reference/request-objects#PaymentMethod):

> This property is required for the CARD payment method.

`PaymentMethodSpecification.tokenizationSpecification`can be optional for `PaymentMethod`, but required for `CARD` payment method type. 

Can we make it optional always?